packages/ckeditor5-ui/src/editorui/accessibilityhelp.ts

```diff
@@ -0,0 +1,382 @@
+	 * that narrow down the context in which the keystrokes are available. Every keystroke category comes with a
+	 * `'common'` group by default.
+	 *
+	 * By default, two categories are registered by the `AccessibilityHelp` plugin: `'contentEditing'` and `'navigation'`.
+	 *
+	 * You can add keystrokes to categories using {@link #registerKeystrokes} method.
+	 */
+	public registerKeystrokeCategory( categoryId: string, definition: { label: string; description?: string } ): void {
+		this._keystrokes.set( categoryId, {
+			...definition,
+			groups: new Map( [ [ DEFAULT_GROUP_ID, { keystrokes: [] } ] ] )
+		} );
+	}
+
+	/**
+	 * Registers keystrokes that will be displayed in the Accessibility help dialog.
+	 *
+	 * To simply register a single keystroke:
+	 *
+	 * ```js
+	 * const t = editor.t;
+	 * editor.plugins.get( 'AccessibilityHelp' ).registerKeystrokes( {
+	 * 	label: t( 'Bold text' ),
+	 * 	keystroke: 'CTRL+B'
+	 * } );
+	 * ```
+	 *
+	 * Please note that by default:
+	 * * two categories are registered by the `AccessibilityHelp` plugin: `'contentEditing'` and `'navigation'`,
+	 * * keystrokes are registered into the `'contentEditing'` category and the `'common'` keystroke group within that category.
+	 *
+	 * To register a keystroke in a specific group in the `'contentEditing'` category:
+	 *
+	 * ```js
+	 * const t = editor.t;
+	 *
+	 * editor.plugins.get( 'AccessibilityHelp' ).registerKeystrokes( {
+	 * 	// Add keystrokes to the "widget" group. If the group does not exist, it will be created.
+	 * 	group: 'widget',
+	 * 	// When creating a new group, you can provide its label.
+	 * 	label: t( 'Keystrokes that can be used when a widget is selected' ),
+	 * 	groupLabel: t( 'Keystrokes that can be used when a widget is selected' ),
+	 * 	keystrokes: [
+	 * 		{
+	 * 			label: t( 'Insert a new paragraph directly after a widget' ),
+	 * 			keystroke: 'Enter'
+	 * 		},
+	 * 		{
+	 * 			label: t( 'Insert a new paragraph directly before a widget' ),
+	 * 			keystroke: 'Shift+Enter'
+	 * 		}
+	 * 	]
+	 * } );
+	 *```
+	 *
+	 * You can add more keystroke categories using {@link #registerKeystrokeCategory} method. To add the keystrokes to the
+	 * `'navigation'` category:
+	 *
+	 * ```js
+	 * this.registerKeystrokes( {
+	 * 	category: 'navigation',
+	 * 	keystrokes: [
+	 * 		{
+	 * 			label: t( 'Close contextual balloons, dropdowns, and dialogs' ),
+	 * 			keystroke: 'Esc'
+	 * 		},
+	 * 		{
+	 * 			label: t( 'Move focus to the visible contextual balloon' ),
+	 * 			keystroke: 'Tab'
+	 * 		}
+	 * 	]
+	 * } );
+	 * ```
+	 */
+	public registerKeystrokes( options: {
+		category?: string;
+		group?: string;
+		groupLabel?: string;
+		keystrokes: Array<AccessibilityHelpKeystrokeDefinition>;
+	} | Array<AccessibilityHelpKeystrokeDefinition> | AccessibilityHelpKeystrokeDefinition ): void {
+		if ( options instanceof Array ) {
+			options = {
+				keystrokes: options
+			};
+		} else if ( typeof options === 'object' && 'keystroke' in options ) {
```

Comment:
I think there is no need to check `typeof options === 'object'` since there is no other possibility due to the first `if` statement.